kuma/landing/jinja2/landing/fellowship.html

```diff
@@ -9,7 +9,7 @@
 
 {% block extrahead %}
 
-  {% set seo_description = _('The Mozilla Developer Network (MDN) Fellowship Program invites world-class web developers to come change the world with Mozilla.') %}
+  {% set seo_description = _('The MDN Web Docs Fellowship Program invites world-class web developers to come change the world with Mozilla.') %}
```

Comment:
Let's not edit this one. The next action on https://developer.mozilla.org/en-US/fellowship should be to convert it to a regular wiki page, not to send new strings to translate. I can do this work in a new PR.